kernel headers fixes this so no point in hacking the crap out of userspace

--- configure
+++ configure
@@ -7743,10 +7743,6 @@
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
-#define u32 int
-#define u64 int
-#define __be32 int
-#define __be64 int
 #include "nbd.h"
 
 int
@@ -7803,10 +7803,6 @@
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
-#define u32 int
-#define u64 int
-#define __be32 int
-#define __be64 int
 #include <linux/nbd.h>
 
 int
--- cliserv.h
+++ cliserv.h
@@ -15,9 +15,6 @@
 #error I need at least some 64-bit type
 #endif
 
-#define __be32 u32
-#define __be64 u64
-
 #ifdef NBD_H_LOCAL
 #include "nbd.h"
 #else
